apps/sms/services/js/conversation/conversation_client.js

```diff
@@ -0,0 +1,79 @@
+/* global bridge,
+          streamClient,
+          Thread
+*/
+
+(function(exports) {
+  'use strict';
+
+  var debug = 0 ?
+    (arg1, ...args) => console.log('[ConversationClient] ' + arg1, ...args):
+    () => {};
+
+  /**
+   * Name of the service that is responsible for managing conversation.
+   * @type {string}
+   */
+  const SERVICE_NAME = 'conversation-service';
+
+  /**
+   * Reference to active bridge client instance.
+   *
+   * @type {Client}
+   */
+  var client;
+
+  /**
+   * ConversationClient is a wrapper around bridge client connected to the
+   * bridge service hosted in a Shared Worker.
+   * @type {Object}
+   */
+  var ConversationClient = {
+    /**
+     * Initialized conversation service client bridge.
+     */
+    init() {
+      var serviceEndpoint = new SharedWorker(
+        '/services/js/conversation/conversation_service.js'
+      );
+
+      serviceEndpoint.port.start();
```

Comment:
I'd rather hide this line inside bridge itself, talked to Wilson, maybe we'll have this or you have some case in mind where we want to control the moment when SharedWorker is started?